addons/sys_core/XEH_PREP.hpp

```diff
@@ -51,4 +52,3 @@ PREP(enableRevealAI);
 PREP(onRevealUnit);
 
 PREP(coreInitPFH);
-//PREP(vehicleCrewPFH);
```

Comment:
?

Reply:
Remnant of an old PFH... it was deprecated with the first push of the new intercom + rack system